app/src/main/java/com/infomaniak/mail/ui/main/settings/SettingRadioButtonView.kt

```diff
@@ -83,4 +89,13 @@ class SettingRadioButtonView @JvmOverloads constructor(
             setImageDrawable(iconDrawable)
         }
     }
+
+    fun setOnClickListener(listener: (() -> Unit)?) {
+        onClickListener = listener
+    }
+
+    fun enable(mustEnable: Boolean) = binding.root.apply {
+        isClickable = mustEnable
+        isEnabled = mustEnable
+    }
```

Comment:
Doesn't seem to work.
Possibly because Realm is flushed and all Signatures are inserted again at each click.
If we remove the API calls _(as discussed in another comment)_, it will possibly resolve this issue.

Reply:
Try to remove `isClickable` I think `isEnabled` should be enough